.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+#### Pre-Flight checklist
+
+- [ ] Screenshots and design review for any changes that affect layout or styling
+  - [ ] Desktop screenshots
+  - [ ] Mobile width screenshots
+  - [ ] Tag @ferdi or @pdpinch for review
+- [ ] Migrations
```

Comment:
Same thing here, I don't think we need the migrations and settings sections.

Reply:
Ah right